schema_editor/app/scripts/views/record/list-controller.js

```diff
@@ -31,8 +31,8 @@
         }
 
         function loadRecordSchema(recordTypeUuid) {
-            RecordTypes.query({ uuid: recordTypeUuid }).$promise.then(function (result) {
-                var recordType = result[0];
+            RecordTypes.get({ id: recordTypeUuid }).$promise.then(function (result) {
+                var recordType = result;
```

Comment:
These fixes address logic bugs affecting the "View records" view in the schema editor, and are unrelated to the frontend.